### DISCUSSION TITLE
Dynamic Scaling for Multiple Agents in Chat Conversations

### DISCUSSION THREAD

**Discussion by ollimacp** [open-webui/open-webui#3272] [General]
Problem:
When multiple agents are selected for a chat conversation, the text becomes too small and difficult to read due to the current static layout.

Solution:
Implement a dynamic scaling feature that allows the size of the content display to be adjusted based on the number of agents participating in the chat. This can be achieved by adding a configuration option in the app settings or preferences, allowing users to toggle between a default and expanded view mode as needed. The feature should automatically adjust the font size or expand the width of the text area when multiple agents are added to the conversation, ensuring that all content remains readable and accessible on any screen size.

Alternatives Considered:
A static layout that doesn't change regardless of the number of agents, which may result in illegible text if multiple agents are selected.

Additional Context:
This feature would improve usability, particularly for users who frequently engage with multiple agents or models at once. It could also be beneficial for users with ultra-wide screens, as it would make better use of available screen space and enhance the overall user experience. This feature should be easily implementable in a front-end web app by adjusting CSS properties based on the number of agents selected.
How its displayed now: 
![image](https://github.com/open-webui/open-webui/assets/33728282/9ce4a291-e98d-4571-8351-57117478b512)

**Comment by tjbck** [comment]
Settings > Interface > Wide screen Mode :)
